common/app/views/support/AmpAd.scala

```diff
@@ -0,0 +1,43 @@
+package views.support
+
+import model.Tag
+import model.Article
+import play.api.libs.json.Json
+
+object AmpAd {
+  private def grabLastFragmentOfId(items: Seq[Tag]) = {
+    items.map { item =>
+      if (item.id == "uk/uk") {
+        item.id
+      } else {
+        val keyword = item.id.split("/").last
+        keyword.replaceAll("""/[+s]+/g""", "-").toLowerCase()
+      }
+    }
+  }
+
+  def buildJsonString(article: Article, uri: String, edition: String): String = {
```

Comment:
could we name it more along the lines of why it's doing it rather than what?  something like `adTargetingJson` if that's what it is?